imagersite/imager_profile/tests.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for the imager_profile app."""
+from django.test import TestCase
+from django.contrib.auth.models import User
+from imager_profile.models import ImagerProfile
+import factory
+
+
+class ProfileTestCase(TestCase):
+    """The Profile Model test runner."""
+
+    class UserFactory(factory.django.DjangoModelFactory):
+        """Makes users."""
+
+        class Meta:
+            """Meta."""
+
+            model = User
+
+        username = factory.Sequence(lambda n: "The Chosen {}".format(n))
+        email = factory.LazyAttribute(
+            lambda x: "{}@foo.com".format(x.username.replace(" ", ""))
+        )
+
+    def setUp(self):
+        """The appropriate setup for the appropriate test."""
+        self.foo = "bar"
+        self.users = [self.UserFactory.create() for i in range(20)]
+
+    def test_profile_is_made_when_user_is_saved(self):
+        """Test profile is made when user is saved."""
+        self.assertTrue(ImagerProfile.objects.count() == 20)
+
+    def test_profile_is_associated_with_actual_users(self):
+        """Test profile is associated with actual users."""
+        profile = ImagerProfile.objects.first()
+        self.assertTrue(hasattr(profile, "user"))
+        self.assertIsInstance(profile.user, User)
+
+    def test_user_has_profile_attached(self):
+        """Test user has profile attached."""
+        user = self.users[0]
+        self.assertTrue(hasattr(user, "profile"))
+        self.assertIsInstance(user.profile, ImagerProfile)
```

Comment:
You need more tests. Test that the model manager you created works correctly. Test that the attributes that you expect are on the model. Test it inside and out so that there's no mystery as to what the model has on it (in terms of attributes) and what it can do.